src/pages/ContactList.js

```diff
@@ -14,6 +15,16 @@ export default function ContactList() {
   return (
     <PageLayout title="Kontaktliste">
       <CardWrapper>
+        {contacts.length ? (
+          <CSVButton>
+          <CSVLinkStyled data={contacts} filename={'my-contacts.csv'}>
+            Kontakte exportieren (CSV)
+          </CSVLinkStyled>
+        </CSVButton>
+        ) : (
+         null
+        )}
+
```

Comment:
I think you could also use the the logical && here
```suggestion
        {contacts.length && (
          <CSVButton>
          <CSVLinkStyled data={contacts} filename={'my-contacts.csv'}>
            Kontakte exportieren (CSV)
          </CSVLinkStyled>
        </CSVButton>
        ) }

```